onnxscript/rewriter/pattern.py

```diff
@@ -169,28 +170,43 @@ class OpPatternBuilder:
 
     def __init__(
         self,
-        opset_pattern: Pattern[str],
-        op_name_pattern: Pattern[str],
+        opset_pattern: OpsetPatternBuilder,
+        op_name: str | Pattern[str],
     ) -> None:
         self.opset_pattern = opset_pattern
-        self.op_name_pattern = op_name_pattern
+        self.op_name = op_name
 
     def __call__(self, *args, **kwargs):
-        # TODO(rama): Unify with convention used elsewhere.
         if "_num_outputs" in kwargs:
```

Comment:
Helpful to a todo to remove this check, maybe after the deprecation period is passed?

Reply:
I think I should remove it ... it was just for my own reminder during the refactoring.